db/contacts copy.json

```diff
@@ -0,0 +1,62 @@
+[
```

Comment:
Możemy usunąć plik `contact.json` oraz `contacts copy.json` - nie będą już więcej używane w aplikacji :)

Reply:
No tak :D To usuwam caly folder db wraz z zawartoscia. Poprawione :D